galsim/config/stamp.py

```diff
@@ -925,7 +925,6 @@ def updateSkip(self, prof, image, method, offset, config, base, logger):
             if image is None:
                 prof = base['wcs'].toImage(prof, image_pos=base['image_pos'])
                 N = prof.getGoodImageSize(1.)
-                N += 2 + int(np.abs(offset.x) + np.abs(offset.y))
```

Comment:
This change seems unrelated? What is going on here?

Reply:
This was never necessary, and while not exactly wrong, it's different from what I do everywhere else to calculate image sizes.  It normally doesn't matter at all, since it's just deciding whether to skip an object, so one extra pixel border rarely matters, and if it (because of this extra pixel) decides not to skip here, it just does a few extra calculations that don't end up adding anything to the image (since the eventual stamp is slightly smaller). 

Anyway, it turned out that with the new random seeds, one of the demo scripts had a difference in the output truth files (not the images -- they were still identical), because the yaml one didn't skip an object, so it made it to the truth file, even though it was fully off the image, but the python one skipped it.